changelogs/unreleased/918-dark64

```diff
@@ -0,0 +1 @@
+Add verifier gadget to stdlib for recursive proof verification
```

Comment:
I don't think we use the term gadget anywhere else. Maybe "Add SNARK verification..." ?